### DISCUSSION TITLE
Welcome to the Maison du Son

### DISCUSSION THREAD

**Discussion by danieleguido** [C2DH/maison-du-son#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect and work together on the *Maison du Son*

**Comment by donsiamese** [comment]
Please add me as a team member. My name is Kirill Mitsurov. My github username is donsiamese

**Reply by Heiphi1997** [reply]
Please add me as a team member. My name is Philippe Hein. My github username is Heiphi1997

**Comment by lausteil** [comment]
Please add me as a team member, my user name is: lausteil. Thanks!

**Comment by MoDzi0907** [comment]
Please add me as a team member. My name is Morgane Denozi. My user name is: MoDzi0907

**Comment by Emily-Griffin** [comment]
Please add me as a team member, my user name is : Emily-Griffin

**Comment by 1AndreRamos** [comment]
Please add me as a team member. My name is André Ramos. My user name is: 1AndreRamos

**Comment by LucaUhrig** [comment]
Hello, I'm Luca. My username is LucaUhrig.

**Comment by hannahschmit** [comment]
Please add me as a team member, my username is hannahschmit.

**Comment by CathyCollard** [comment]
Please add me as a team member, my user name is: CathyCollard

**Comment by Alice-PR** [comment]
Please add me as a team member. My user name is: Alice-PR. Thank you.

**Comment by celiahamissi** [comment]
Please add me as a team member, my user name is : celiahamissi

**Comment by KevinFernandesX** [comment]
Please add me as a team member. My name is Kevin Fernandes Pereira, My user name is: KevinFernandesX

**Reply by CemAr2910** [reply]
Please add me as a team member. My name is Armin Ceman and my username is: CemAr2910

**Comment by Benji1510** [comment]
Please add me as a team member. My name is Benjamin Juchem and my user name is : Benji1510

**Comment by LaurenceHarf** [comment]
Please add me as a team member. My name is Laurence Harf. My github username is LaurenceHarf

**Comment by YassineZaouch** [comment]
Hello. I'm ZAOUCH Yassine. My username is YassineZaouch.

**Comment by christelletimis** [comment]
Please add me as a team member. My name is Christelle Timis. My username is christelletimis.

**Comment by Schlottert4** [comment]
Please add me as a team member. My name is Pol Schlottert. My user name is: Schlottert4

**Comment by Pampelmann007** [comment]
Please add me as a team member. My name is Andreas Fickers. My username is AndreasFickers

**Comment by caroresch99** [comment]
Hello, I'm Caroline Resch, my username is caroresch99
